ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Eras.hs

```diff
@@ -133,7 +126,10 @@ type EraCrypto era = Crypto era
 -- needed to determine the hard fork point. In the future this should be
 -- replaced with an appropriate API - see
 -- https://github.com/input-output-hk/ouroboros-network/issues/2890
-class ( SL.ShelleyBasedEra era
+class ( Core.EraSegWits era
+      , SL.ApplyTx era
+      , SL.ApplyBlock era
+      , SL.CanStartFromGenesis era
```

Comment:
We confirmed on a call with Ledger that their refactoring replaced `ShelleyBasedEra` by these constraints.